R/pkg/inst/worker/worker.R

```diff
@@ -36,7 +36,14 @@ compute <- function(mode, partition, serializer, deserializer, key,
       # available since R 3.2.4. So we set the global option here.
       oldOpt <- getOption("stringsAsFactors")
       options(stringsAsFactors = FALSE)
-      inputData <- do.call(rbind.data.frame, inputData)
+
+      # Handle binary data types
+      if ("raw" %in% sapply(inputData[[1]], class)) {
+        inputData <- SparkR:::rbindRaws(inputData)
```

Comment:
When I remove `SparkR:::` and run the tests locally I see error with `rbindRaws not found`. 

So it looks like the `SparkR:::` needs to be there to access private functions in SparkR from the worker node.